Airports/EGNH/SMR/Geo.txt

```diff
@@ -95,6 +95,16 @@ N053.46.41.706 W003.02.15.437 N053.46.41.616 W003.02.15.992 smrWhite
 N053.46.41.616 W003.02.15.992 N053.46.40.372 W003.02.23.058 smrWhite
 N053.46.40.372 W003.02.23.058 N053.46.41.507 W003.02.23.658 smrWhite
 
+; - Main_Apron_Boundary
```

Comment:
```suggestion
;Main_Apron_Boundary
```
I usually replace all `; - ` with just `;`, for tidyness and consistency with the rest of the file.